apps-rendering/src/components/shared/relatedContent.tsx

```diff
@@ -84,6 +85,22 @@ const listStyles = css`
 	}
 `;
 
+const styles = css`
+	border-top: 1px solid ${neutral[46]};
+	padding-top: ${remSpace[3]};
+	padding-left: ${remSpace[4]};
+	padding-right: ${remSpace[4]};
+
+	${until.wide} {
+		margin-left: -${remSpace[4]};
+		margin-right: -${remSpace[4]};
```

Comment:
What do the negative margins do here?

Reply:
Good question. The related content `<section>` now has a visible `top-border`.
This is all well and good at wider breakpoints, but at narrow breakpoints (i.e. on a phone, which is where most people would see this content), there would be a visible margin in the areas circled in red in the screenshot below. 

![image](https://user-images.githubusercontent.com/57295823/151543981-43caff3f-5e22-43dd-974f-9be4ff8fbe44.png)

Adding negative margin and offsetting it with positive padding allows the border to extend across the screen at smaller breakpoints, while keeping the content of the section in the same position.